business/istio_config.go

```diff
@@ -652,7 +701,38 @@ func getPermissions(k8s kubernetes.ClientInterface, namespace, objectType string
 			log.Errorf("Error getting permissions [namespace: %s, api: %s, resourceType: %s]: %v", namespace, api, resourceType, permErr)
 		}
 	}
-	return canCreate, (canUpdate || canPatch), canDelete
+	return canCreate, canPatch, canDelete
+}
+
+func getPermissionsApi(k8s kubernetes.ClientInterface, namespace, api string) (bool, bool, bool) {
+	var canCreate, canPatch, canDelete bool
+	/*
+		Kiali only uses create,patch,delete as WRITE permissions
+
+		"update" creates an extra call to the API that we know that it will always fail, introducing extra latency
+
+		Synced with:
+		https://github.com/kiali/kiali-operator/blob/master/roles/default/kiali-deploy/templates/kubernetes/role.yaml#L62
```

Comment:
just FYI - this link will eventually grow stale because its linking to "master" rather than a fixed tag. Probably should have used "v1.36.0".